src/instruments/src/Common/gauges.tsx

```diff
@@ -171,3 +174,119 @@ export const VerticalSegment: FC<VerticalSegmentProps> = ({ x, y, height, rangeS
         />
     );
 };
+
+export function splitDecimals(value, type) {
+    if (type === 'oil') {
+        value = value * 0.01 * 25;
+    } else if (type === 'vib') {
+        value = value < 0 ? 0.0 : value;
+    }
+    const decimalSplit = value.toFixed(1).split('.', 2);
+    return (decimalSplit);
+}
+
+export function valueRadianAngleConverter(value, min, max, endAngle, startAngle) {
+    const valuePercentage = (value - min) / (max - min);
+    let angleInRadians = startAngle > endAngle
+        ? startAngle + (valuePercentage * (360 - startAngle + endAngle)) - 90
+        : startAngle + (valuePercentage * (endAngle - startAngle)) - 90;
+    angleInRadians *= (Math.PI / 180.0);
+    return ({
+        x: Math.cos(angleInRadians),
+        y: Math.sin(angleInRadians),
+    });
+}
+
+type GaugeMarkerComponentType = {
```

Comment:
**issue:** if we're going to add components here, then they should be removed from the other pages using them as part of this PR.

Reply:
It's fine, I'll remove them and add them to the Pressure page.

Reply:
I mean that they are fine in `gauges.tsx`, but then `Crz.ts` for example should use those and not its own.

Reply:
Oh I see. Yeah, let me sort the Cruise page to use this gauge component